docs/section23/Rule23-1.md

```diff
@@ -36,15 +36,15 @@
 - get the hvac system from the hvac_system_id: `hvac_system = get_component_by_id(hvac_system_id)`
 - get the heating system: `heating_system = hvac_system.heating_system`
 - get the heatpump_low_shutoff_temperature = `heatpump_low_shutoff = heating_system.heatpump_low_shutoff_temperature`
+- get the auxiliary heat high temperature shutoff = 'aux_heat_high_temp_shutoff = heating_system.heatpump_auxilliary_heat_high_shutoff_temperature`
+- get the auxiliary heat energy source: `aux_heat_energy_source = heatpump_auxilliary_heat_type`
```

Comment:
I think this needs to be modified to `aux_heat_energy_source = **heating_system.**heatpump_auxilliary_heat_type`